lib/ZumoOta/src/WebServerCustom.cpp

```diff
@@ -58,7 +59,7 @@
  * Local Variables
  *****************************************************************************/
 Upload upload;
-
+MySettings set;

```

Comment:
"set" has a diifferent meaning than "setting".  Why not call it "settings" ?  
Are settings only needed here  ?
Should they be a static? If they depend on file system, you need to init them after the file system

Reply:
The reason for renaming it here is that Mysettings was already defined within NetworkCustom.